### DISCUSSION TITLE
Discord channel.

### DISCUSSION THREAD

**Discussion by haydermabood** [rimbu-org/rimbu#196] [Ideas]
Hello, and thanks for your great efforts, I very much like this library!

Among other possible things that would increase and enhance the activity around Rimbu; is creating a Discord channel. I hope that you would consider this suggestion.

**Comment by vitoke** [comment]
Hey, thanks for your kind words. I think creating a Discord server is a great idea. I do not have a lot of experience with it, but I've tried to set something up.

Please (everyone) join and let me know if you have any improvement suggestions:

[Rimbu Lounge](https://discord.gg/J74vkceDFh)

I'll add links to the documentation soon.

**Reply by scarf005** [reply]
hi, looks like the link is invalid. could you confirm that whether discord is still open or is closed?

**Reply by vitoke** [reply]
Wow that was embarassing :) Apparently it was a temporary invitation. Yes the Discord is open but I guess partly to the mistake not very active yet. Feel welcome though!
